hw2/05-form.js

```diff
@@ -1,15 +1,43 @@
 // Add your code here
-const userName = document.getElementById('InputName').value
 const submit = document.getElementById('Submit');
 const reset = document.getElementById('Reset');
 
+submit.addEventListener('click', ()=>{
 
-console.log(userName)
+    const userName = document.getElementById('InputName').value
+    const email = document.getElementById('InputEmail').value
+    const registration = document.getElementById('Registration').value
+    const progCheck = document.getElementById('ProgrammingLang').checked
+    const opCheck = document.getElementById('OperatingSystems').checked
+    const fullCheck = document.getElementById('FullStack').checked
+    const anything = document.getElementById('Anything').value
+    console.log(userName)
+    console.log(email)
+    console.log(registration)
 
-submit.addEventListener('click', ()=>{
+    if(progCheck === true){
+        console.log("Programming Languages")
+    }
+    if(opCheck === true){
+        console.log("Operating Systems")
+    }
+    if(fullCheck === true){
```

Comment:
Adding an array for courses would really help your program output look better. You can create new array and if the course is checked, that can be pushed to this array. While displaying the output, array name can give the desired output.

Reply:
That makes sense, could help make this a little more condensed